src/SprykerEco/Zed/Payone/Communication/Controller/TransactionController.php

```diff
@@ -26,7 +27,7 @@ class TransactionController extends AbstractController
      *
      * @return \Symfony\Component\HttpFoundation\StreamedResponse
      */
-    public function statusUpdateAction(Request $request)
+    public function statusUpdateAction(Request $request): StreamedResponse
```

Comment:
It's a part of public API, please remove added type hints.

Reply:
Fixed.